src/CoreBundle/Service/PortalDomainServiceInterface.php

```diff
@@ -35,7 +35,7 @@ public function getActiveDomain();
     public function getActivePortal();
 
     /**
-     * Returns the first defined portal.
+     * Returns the primary portal (cms config).
```

Comment:
There is a document in which suche changes should be added, is there not?

Reply:
Not for 6.2...